apps/examples/rdp/rdp_main.c

```diff
@@ -60,7 +60,8 @@
 
 #include <tinyara/config.h>
 #include <tinyara/arch.h>
-#include <arch/chip/amebad_nsc.h>
+
+#include <tinyara/tz_context.h>
```

Comment:
Can you make app change as separate commit?

Reply:
ok

Reply:
done